### PR TITLE
fix some typo.

### PR DESCRIPTION
**Pull request by paul-cheung**
## References
N/A

## PR Checklist
* [x] Closes nothing
* [x] CLA signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/Terminal) and sign the CLA
* [x] ~~Tests added/passed~~
* [x] ~~Requires documentation to be updated~~
* [x] I've discussed this with core contributors already. If not checked, I'm ready to accept this work might be rejected in favor of a different grand plan.

## Detailed Description of the Pull Request / Additional comments
just fix some typo in documentations

### CODE REVIEW

File: doc/cascadia/Keybindings-spec.md [hunk 1]
```diff
@@ -1,108 +1,108 @@
...
+# Keymapping spec
+
+* author: Mike Griese __migrie__
+* created on: 2018-Oct-23
+
+## Abstract
+It should be possible to configure the terminal so that it doesn't send certain keystrokes as input to the terminal, and instead triggers certain actions. Examples of these actions could be copy/pasting text, opening a new tab, or changing the font size.
+
+This spec describes a mechanism by which we could provide a common implementation of handling keyboard shortcuts like these. This common implementation could then be leveraged and extended by the UX implementation as to handle certain callbacks in the UX layer. For example, The TerminalCore doesn't have a concept of what a tab is, but the keymap abstraction could raise an event such that a WPF app could implement creating a new tab in its idomatic way, and UWP could implement them in their own way.
```
Comment (miniksa) [inline doc/cascadia/Keybindings-spec.md]: Your merge appears to have flattened this fix.

```suggestion
This spec describes a mechanism by which we could provide a common implementation of handling keyboard shortcuts like these. This common implementation could then be leveraged and extended by the UX implementation as to handle certain callbacks in the UX layer. For example, The TerminalCore doesn't have a concept of what a tab is, but the keymap abstraction could raise an event such that a WPF app could implement creating a new tab in its idiomatic way, and UWP could implement them in their own way.
```

File: doc/building.md [hunk 2]
```diff
@@ -29,4 +29,4 @@ Openconsole has three configuration types:
 - Release
 - AuditMode
 
-AuditMode is an experimental mode that enables some additional static analyis from CppCoreCheck.
```
Comment (miniksa) [inline doc/building.md]: You appear to have lost this one with the merge too.

File: doc/submitting_code.md [hunk 3]
```diff
@@ -20,7 +20,7 @@ Sometimes VSTS doesn't want to allow a cherry pick to the inbox branch. It might
 1. make sure you have pulled the latest commits for the `dev/main` and `inbox` branches
 2. make a new branch from inbox
 3. cherry-pick the commits from the PR to the newly created branch (this is easier if you squashed your commits when you merged into `dev/main`
-4. fix any merge conficts and commit
```
Comment (miniksa) [inline doc/submitting_code.md]: This file's changes also look lost on merge.

**Comment by paul-cheung** [comment]
> This one looks lost
> 
> EDIT: sorry for the weird double-change request.

**Comment by paul-cheung** [comment]
@carlos-zamora @miniksa , okay, i will raise another PR.

**Pull request closed without merging**